.github/workflows/verify.yml

```diff
@@ -1,5 +1,21 @@
 name: Verify
 
+# https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions
+permissions:
+  actions: none
+  checks: none
+  contents: none
+  deployments: none
+  id-token: none
+  issues: none
+  discussions: none
+  packages: none
+  pages: none
+  pull-requests: none
```

Comment:
Pretty sure this should have read access here.

Reply:
There's no programmatic interaction with the pull requests API in this job, so shouldn't be needed :+1:

Custom fork for verification:
<img width="872" alt="image" src="https://user-images.githubusercontent.com/60357436/156236842-124bd79c-5dbf-4ef5-bf6f-d891d6b8deab.png">